dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -453,38 +454,73 @@ def matrix_rank(A, tol=None, hermitian=False):
 
 def multi_dot(arrays, out=None):
     """
-    Compute the dot product of two or more arrays in a single function call
+    Compute the dot product of two or more arrays in a single function call.
+
+    For full documentation refer to :obj:`numpy.multi_dot`.
 
     Parameters
     ----------
-    arrays : sequence of array_like
+    arrays : sequence of dpnp.ndarray or usm_ndarray
         If the first argument is 1-D it is treated as row vector.
         If the last argument is 1-D it is treated as column vector.
         The other arguments must be 2-D.
-    out : ndarray, optional
-        unsupported
+    out : {None, dpnp.ndarray, usm_ndarray}, optional
+        Output argument. This must have the exact kind that would be returned
+        if it was not used. In particular, it must have the right type, must be
+        C-contiguous, and its dtype must be the dtype that would be returned
+        for `dot(a, b)`. If these conditions are not met, an exception is
+        raised, instead of attempting to be flexible.
 
     Returns
     -------
-    output : ndarray
+    out : dpnp.ndarray
         Returns the dot product of the supplied arrays.
 
     See Also
     --------
-    :obj:`numpy.multi_dot`
+    :obj:`dpnp.dot` : Returns the dot product of two arrays.
+    :obj:`dpnp.inner` : Returns the inner product of two arrays.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> from dpnp.linalg import multi_dot
+    >>> A = np.random.random((10000, 100))
+    >>> B = np.random.random((100, 1000))
+    >>> C = np.random.random((1000, 5))
+    >>> D = np.random.random((5, 333))
+
+    the actual dot multiplication
+
+    >>> multi_dot([A, B, C, D]).shape
+    (10000, 333)
+
+    instead of
+
+    >>> np.dot(np.dot(np.dot(A, B), C), D).shape
+    (10000, 333)
+
+    or
+
+    >>> A.dot(B).dot(C).dot(D).shape
+    (10000, 333)
 
     """
 
-    n = len(arrays)
+    dpnp.check_supported_arrays_type(*arrays)
 
-    if n < 2:
-        checker_throw_value_error("multi_dot", "arrays", n, ">1")
+    if out is not None:
```

Comment:
It is duplicated and we do not even need it inside `dpnp_multi_dot` because there also `dpnp.dot` is called at the end which checks for out keyword to be correct.

Reply:
Does it mean that the check can be safely removed?

Reply:
yes